### ISSUE TITLE
TensorRT engine binding error

### ISSUE THREAD

**Issue by Davidnet** [tensorflow/tensorflow#23165]
*System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): No.
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Jetson TX2, Linux4Tegra Xenial 16.04
- TensorFlow installed from (source or binary): Source
- TensorFlow version (use command below): 1.8.0
- Python version: 2.7.12
- Bazel version (if compiling from source): 0.18.0
- GCC/Compiler version (if compiling from source): gcc5
- CUDA/cuDNN version: 9.0/7.1.5
- GPU model and memory: Jetson tx2 8GB

You can collect some of this information using our environment capture [script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh)
You can also obtain the TensorFlow version with
python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"

**Describe the current behavior**:

I have frozen a graph that contains the following operations:

* tf.layers.conv2d(*args, **kwargs)
* tf.layers.batch_normalization(net, **batch_norm)

I will try to provide a minimal graph that have this problem.
**Describe the expected behavior**:
I create a trt_graph by using the function trt.create_inference_graph and it creates a graph successfully  but whenever I try to make an inference I encounter:

```
2018-10-22 15:29:17.537843: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1053] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with│
 363 MB memory) -> physical GPU (device: 0, name: NVIDIA Tegra X2, pci bus id: 0000:00:00.0, compute capability: 6.2)                                            │
2018-10-22 15:29:20.426303: F tensorflow/contrib/tensorrt/shape_fn/trt_shfn.cc:76] TensorRT engine cannot find binding: ModelBase/Conv2dBatchNorm/Relu     
Aborted (core dumped)
```
Any guideline on how to provide a better log issue?

Thanks

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Exact command to reproduce
Mobile device

**Comment by Davidnet** [comment]
Hi @ymodak I've attached a small minimal module that reproduces the problem in the Jetson TX2, let me know if I can be of help.

There are two python modules one that creates the tensorrt graph and serializes it. the other try to run inference on an image. .

[tensorrt_problem.tar.gz](https://github.com/tensorflow/tensorflow/files/2516567/tensorrt_problem.tar.gz)

**Comment by Davidnet** [comment]
We have encounter other issues with performance on jetson tx2 while using (on house compiling) tensorflow 1.10 or 1.11, do you provide wheels for python2.7 for the jetson tx2.

**Comment by Davidnet** [comment]
Any updates?

**Comment by samikama** [comment]
Hi @Davidnet,
Did you check
https://docs.nvidia.com/deeplearning/dgx/install-tf-jetsontx2/index.html ?

**Comment by samikama** [comment]
We are working on producing the packages.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @samikama: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by samikama** [comment]
I am closing this since the update will come with the next pip package for TX2.
